client/app/vps/modal/shortcut/reboot/vps-reboot.controller.js

```diff
@@ -0,0 +1,53 @@
+class VpsRebootCtrl {
+    constructor ($translate, $uibModalInstance, CloudMessage, serviceName, VpsService) {
+        this.$translate = $translate;
+        this.$uibModalInstance = $uibModalInstance;
+        this.CloudMessage = CloudMessage;
+        this.serviceName = serviceName;
+        this.VpsService = VpsService;
+
+        this.loader = {
+            init: false,
+            save: false
+        };
+        this.model = {};
+        this.selected = {
+            rescue: false
+        };
+    }
+
+    $onInit () {
+        this.loader.init = true;
+        this.VpsService.getTaskInError(this.serviceName)
+            .then (tasks => { this.loadVpsRescueMode(tasks) });
```

Comment:
no .finally(() => { this.loader.init = false });?

Reply:
the finally is in the sub function.